cves/CVE-2012-6617.yml

```diff
@@ -188,8 +186,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: You would need domain specific knowledge to get the vector that would cause the denial of service.
```

Comment:
This vulnerability could be caught by an automated tool. Keep in mind that developers writing an automated tool to discover vulnerabilities typically have domain specific knowledge of the system they are developing the tool for.

Reply:
This statement "These are tools that require little knowledge of the domain," Gave me the answer as a false